src/routes/router.js

```diff
@@ -30,6 +30,9 @@ import configureStore from './middleware'
 
 import * as segment from 'servers/segment/index'
 
+const ORGANIZATION_TAB_REGEX = /tab=(\w+)/
```

Comment:
that's just part of the regex. It is just checking the `tab=(\w+)` plus bit.. Also, this is just included for development, so hot reloading doesn't change tabs. Sorry for that not being clear, that was present in the original PR description. I was planning on removing this in prod as it is hacky.